src/main/twirl/gitbucket/core/account/ssh.scala.html

```diff
@@ -19,7 +19,7 @@
     </div>
     <form method="POST" action="@context.path/@account.userName/_ssh" validate="true" autocomplete="off">
       <div class="panel panel-default">
-        <div class="panel-heading strong">Add a SSH Key</div>
+        <div class="panel-heading strong">Add an SSH public key</div>
```

Comment:
```suggestion
        <div class="panel-heading strong">Add a SSH public key</div>
```
As @McFoggy says, this would be better.